diode-server/reconciler/ingestion_processor_internal_test.go

```diff
@@ -81,6 +83,11 @@ func TestWriteIngestionLog(t *testing.T) {
 			p := &IngestionProcessor{
 				redisClient: mockRedisClient,
 				logger:      slog.New(slog.NewJSONHandler(os.Stdout, &slog.HandlerOptions{Level: slog.LevelDebug, AddSource: false})),
+				Config: Config{
+					AutoApplyChangesets:        true,
```

Comment:
Should we add a test with apply false to check outcome?

Reply:
Good point! Thank you